CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.3.0] - 2024-10-03
+
+### Added
+
+- Implement AbortMultipartUpload operation (109).
+
+### Changed
+
+- Improve performance of Multipart Uploads (114).
+- Improve debug log messages (124).
+- Improve CMake (irods/irods 6251, irods/irods 6256, irods/irods 7265).
```

Comment:
will you put #'s in here... and in the rest of the issues?

Reply:
Yes. Wanted people to see it before doing that.

Reply:
gotcha - i think it's fine for now - these won't 'pollute' the issues i don't think...

Reply:
Added pounds to changelog content.